### PR TITLE
MCP-1622: Delete old demo code

### PR DESCRIPTION
**Pull request by dimitri-amida**
# Delete old demo code

### What was the problem?

There is a lot of unused code built for demo purposes that's cluttering the app and slowing down the build.

### How does this fix it?

Deleted old demo code

### Jira Tickets

- [MCP-1622](https://amida.atlassian.net/browse/MCP-1622)

## How to test this PR

- Simply regression test endpoints to make sure nothing is broken

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

**Review by wlin2019 — APPROVED** [review]
LGTM

File: controller/src/main/java/gov/va/vro/controller/demo/mapper/AssessHealthDataRequestMapper.java [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package gov.va.vro.controller.demo.mapper;
```
[the remark this answers is not in the document]
Comment (dimitri-amida) [reply controller/src/main/java/gov/va/vro/controller/demo/mapper/AssessHealthDataRequestMapper.java]: @wlin2019 you are right, I had missed that. I removed this demo code as well

**Review by ianwolson — APPROVED** [review]
This looks much better thank you for cleaning this up!

**Review by wlin2019 — APPROVED** [review]
LGTM

**Pull request merged**